config/v1/types_infrastructure.go

```diff
@@ -354,6 +354,17 @@ type AWSPlatformSpec struct {
 	// There must be only one ServiceEndpoint for a service.
 	// +optional
 	ServiceEndpoints []AWSServiceEndpoint `json:"serviceEndpoints,omitempty"`
+
+	// ResourceTags is a list of additional tags to apply to AWS resources created for the cluster.
+	// See https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html for information on tagging AWS resources.
+	// AWS supports a minimum of 10 tags per resource. OpenShift reserves 5 tags for its use, leaving 5 tags
+	// available for the user.
+	// ResourceTags field is mutable and items can be removed.
+	// When a tag is removed from this list, management of the tag is stopped.
+	// The tag will remain in an unmanaged state on any existing resource.
+	// +kubebuilder:validation:MaxItems=5
+	// +optional
+	ResourceTags []AWSResourceTag `json:"resourceTags,omitempty"`
```

Comment:
Just re-looking at this, we didn't add a new type here, we discussed on the enhancement that this should be done so that the spec and status can evolve separately.

Can you make sure the PR here reflects [the enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/api-review/custom-tags-aws.md#api-extensions) which already contains a reviewed Go patch